app/.env.example

```diff
@@ -1,4 +1,4 @@
 SERVER_URL=http://localhost:6060
```

Comment:
What's `SERVER_URL` used for? Can this be deleted?

Reply:
Good point. Deleted in [f232d57](https://github.com/hackforla/HomeUniteUs/pull/637/commits/f232d57183d9b59542a4beea66de09a44161fa05).